src/pup/plugins/pip_install.py

```diff
@@ -25,13 +26,33 @@ def usable_in(ctx):
         )
 
     def __call__(self, ctx, dsp):
+        python_path = str(ctx.python_runtime_dir / ctx.python_rel_exe)
+
+        platform_flags = []
+        if ctx.pip_platform:
+            platform_flags.append('--platform={}'.format(ctx.pip_platform))
+            platform_flags.append('--only-binary=:all:')
+            # TODO: This should probably be done with the spawn helpers, but
+            #       done this way as a proof of concept
+            site_packages_path = subprocess.check_output([
```

Comment:
Not sure about this check: what's the objective?

Reply:
It's not specifically a check, but it is trying to retrieve the "site_packages" path, this path is needed to use the `--target` flag, which is needed if we want to use the `--platform` flag.